src/v2/System/Server/shortcuts.ts

```diff
@@ -0,0 +1,23 @@
+import express, { NextFunction } from "express"
+import { getENV } from "v2/Utils/getENV"
+import { ArtsyRequest, ArtsyResponse } from "lib/middleware/artsyExpress"
+
+export const app = express()
+
+app.get(
+  "/:short",
+  (req: ArtsyRequest, res: ArtsyResponse, next: NextFunction) => {
+    const short = req.params.short.toLowerCase()
+
+    fetch(`${getENV("API_URL")}/api/v1/shortcut/${short}`, {
+      headers: {
+        "X-XAPP-TOKEN": getENV("ARTSY_XAPP_TOKEN"),
+      },
+    })
+      .then(response => response.json())
+      .then(data => {
+        res.redirect(301, data.long)
```

Comment:
Previously we weren't 301ing. I'm not if this is desirable since you do technically have the ability to update a redirect. cc @jonallured

Reply:
Did some poking around in the db here:

https://artsy.slack.com/archives/C01ADJNCS5D/p1650317555473629

My take is: let's 301. 😃